### ISSUE TITLE
[TW-1841] Error creating task because text like 40th is interpreted as a date

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1857]
_Steve Kroon on 2016-08-12T06:28:48Z says:_ 

 Example of the error:

$ task add Buy 40th birthday gift
Error: no month has more than 31 days

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2016-08-12T06:28:48Z
Modified: 2016-08-12T11:44:26Z
```

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2016-08-12T11:44:26Z says:_ 

 This was fixed in version 2.5.1.
